### PR TITLE
Meeting summaries for 2016-12-22

### PR DESCRIPTION
**Pull request by G1lius**

### CODE REVIEW

File: _posts/en/meetings/2016-12-22-meeting.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+title: IRC meeting summary for 2016-12-22
+permalink: /en/meetings/2016/12/22/
+name: 2016-12-22-meeting
+type: meetings
+layout: page
+lang: en
```
Comment (btcdrak) [inline _posts/en/meetings/2016-12-22-meeting.md]: looks like some spurious control characters.

Comment (G1lius) [inline _posts/en/meetings/2016-12-22-meeting.md]: Yeah, I don't know what it is. It's not in the source file.

It's also on all the other meetings, I assume something new from github. Don't know what it means or how to remove it though.

Comment (MarcoFalke) [inline _posts/en/meetings/2016-12-22-meeting.md]: I think it is just the Windows way of saying "new line".

Comment (G1lius) [inline _posts/en/meetings/2016-12-22-meeting.md]: It is indeed. Very weird, couldn't fix it with a new commit from my linux machine either so I made a new PR for this.

**Pull request closed without merging**